packages/qwik/src/core/components/prefetch.ts

```diff
@@ -140,21 +144,18 @@ export const PrefetchGraph = (
 ) => {
   const serverData = useServerData<Record<string, string>>('containerAttributes', {});
   const resolvedOpts = {
+    // /build/q-bundle-graph-${manifestHash}.json is always within the q:base location /build/
     base: serverData['q:base'],
     manifestHash: serverData['q:manifest-hash'],
     scope: '/',
     verbose: false,
     path: 'qwik-prefetch-service-worker.js',
     ...opts,
   };
-  const args = [
-    'graph-url',
-    resolvedOpts.base,
-    resolvedOpts.base + `q-bundle-graph-${resolvedOpts.manifestHash}.json`,
-  ]
+  const args = ['graph-url', resolvedOpts.base, `q-bundle-graph-${resolvedOpts.manifestHash}.json`]
     .map((x) => JSON.stringify(x))
     .join(',');
-  const code = `(window.qwikPrefetchSW||(window.qwikPrefetchSW=[])).push(${args})`;
+  const code = `(window.qwikPrefetchSW||(window.qwikPrefetchSW=[])).push([${args}])`;
```

Comment:
you can just stringify all of `args`

Reply:
oh yeah haha this is abusing how array is serialized